.github/workflows/component-bumper.yml

```diff
@@ -7,17 +7,20 @@ on:
 jobs:
 
   build:
-    name: CNAO Component Bump Job
+    name: [master] CNAO Component Bump Job
     runs-on: ubuntu-latest
     steps:
 
+    - name: Set environment variables
+      run: echo "::set-env name=BASE_BRANCH::master";
+
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
       with:
-        ref: master
+        ref: ${{ env.BASE_BRANCH }}
 
-    - name: Pull latest master
-      run:  git pull --ff-only --rebase origin master
+    - name: Pull latest of latest branch
+      run:  git fetch origin ${{ env.BASE_BRANCH }} && git pull --ff-only --rebase origin ${{ env.BASE_BRANCH }}
```

Comment:
Aren't  this two actions redundant ? we are checking out BASE_BRANCH already in the first action

Reply:
you are correct.
Done